docs/docs/installation/kubernetes.mdx

```diff
@@ -157,7 +157,7 @@ The following example installs the drivers for BigQuery and Elasticsearch, allow
 ```yaml
 bootstrapScript: |
   #!/bin/bash
-  pip install psycopg2==2.9.6 \
+  pip install psycopg2-binary==2.9.10 \
```

Comment:
I would recommend `pip install .[postgres]` here to align with the version we use in CI and generally recommend to the community. I'm not sure why it's pinned to `2.9.6` ATM though there's probably a good reason for it.

This goes for subsequent lines as well (referencing the setuptools extra_requires packages). Though it's challenging to keep all the packages up-to-date and all. You may want to run a newer / older version of all those drivers for whatever reason applies in your env.

https://github.com/apache/superset/blob/master/pyproject.toml#L161

Reply:
I'll test this and update PR if it works.

Reply:
Works, updated instructions to refer to pyproject.toml.